ansible/ansible-risk-insight
# PR to push implementation Inline replace functionality from ARI

PR to push implementation Inline replace functionality from ARI.

Currently the `--fix` operation is supported/enabled with `--scan-per-target` command.

Steps to test:
- Pull the PR changes
- Run the ARI project command as below:
`ari project <local github repository> --out-dir /tmp/test --save-only-rule-result --scan-per-target --task-num-threshold 100 --fix`
- This should update the input local github repository, with all the ARI suggested mutation by performing an inline replace functionality.
